DAO/project/fuel-toolchain.toml

```diff
@@ -1,6 +1,6 @@
 [toolchain]
-channel = "latest-2023-11-20"
+channel = "nightly-2023-12-06"
```

Comment:
I assume this is temporary until the release that you need is out

Reply:
Eventually this will be beta-5